src/SDKs/Search/Management/Management.Search/generate.ps1

```diff
@@ -1 +1,33 @@
-Start-AutoRestCodeGeneration -ResourceProvider "search/resource-manager" -AutoRestVersion "latest"
+# ---------------------------------------------------------------------------------- 
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# ---------------------------------------------------------------------------------- 
+
+<#
+
+.SYNOPSIS
+    Powershell script that generates the C# SDK code for Management.Search from a Swagger spec
+
+.DESCRIPTION
+    This script:
+    - fetches the config file from user/branch provided
+    - Generates code based off the config file provided
+
+.PARAMETER SpecsRepoFork
+    The Rest Spec repo fork which contains the config file; the default is Azure.
+
+.PARAMETER SpecsRepoBranch
+    The Branch which contains the config file; the default is master.
+
+#>
+
+Param(
+    [string] $SpecsRepoFork = "Azure",
+    [string] $SpecsRepoBranch = "master"
+)
+
+$repoRoot = "$PSScriptRoot\..\..\..\..\.."
+$generateFolder = "$PSScriptRoot\Generated"
+
+Start-AutoRestCodeGeneration -ResourceProvider "search/resource-manager" -AutoRestVersion "latest" -SpecsRepoFork $SpecsRepoFork -SpecsRepoBranch $SpecsRepoBranch
```

Comment:
When running Start-AutoRestCodegeneration against a private fork, I had to point -SdkRepoRootPath at the src directory ($PSscriptRoot\..\..\). Can you add this? It's not obvious what to pass here.